tests/structured_tests/parse_regex.toml

```diff
@@ -0,0 +1,29 @@
+query = """db-1 response | parse regex "in (?P<duration>\\\\d+)ms" """
```

Comment:
that is a lot of escaping! (But is it just toml that we're escaping here?) The docs seem to indicate that those escapes wouldn't be required.

Reply:
It's escaping for toml.  Looks like I can change it to a literal string (single-quote instead of double)